components/cosmetic_filters/renderer/cosmetic_filters_js_handler.h

```diff
@@ -72,6 +72,7 @@ class CosmeticFiltersJSHandler {
   void InjectStylesheet(const std::string& stylesheet, int id);
   void UninjectStylesheet(int id);
 
+  bool generichide_;
```

Comment:
it's strange that lint doesn't give any error about that uninitialised class member.